x/pairing/keeper/pairing_test.go

```diff
@@ -893,3 +893,27 @@ func TestNoRequiredGeo(t *testing.T) {
 		require.NotEqual(t, freePlanPolicy.GeolocationProfile, provider.Geolocation)
 	}
 }
+
+// TestGeoSlotCalc checks that the calculated slots always hold a single bit geo req
+func TestGeoSlotCalc(t *testing.T) {
+	minStake := sdk.NewInt(0)
+	geoReqName := pairingscores.GeoReq{}.GetName()
+
+	// iterate over all possible geolocations, create a policy and calc slots
+	for i := 0; i <= int(planstypes.Geolocation_GL); i++ {
+		policy := planstypes.Policy{GeolocationProfile: uint64(i)}
```

Comment:
how is this working?
you never set the number of providers in the policy

Reply:
you're right. It passed because it basically checked nothing. fixed it. see https://github.com/lavanet/lava/commit/9e7d567b71d64b426c40f489d9a0029f0e240ab6